lrpar/cttests/src/lib.rs

```diff
@@ -355,16 +355,15 @@ fn test_expect() {
 #[test]
 fn test_grmtools_section_files() {
     use glob::glob;
-    use std::env;
     use std::fs::File;
     use std::io::BufReader;
     use std::io::{BufRead as _, Read as _};
 
-    let manifest_dir = env::var("CARGO_MANIFEST_DIR").unwrap();
+    let manifest_dir = env!("CARGO_MANIFEST_DIR");
     let examples_glob = format!("{manifest_dir}/../examples/**");
     let examples_l_glob = format!("{examples_glob}/*.l");
     let examples_y_glob = format!("{examples_glob}/*.y");
-    let out_dir = env::var("OUT_DIR").unwrap();
+    let out_dir = env!("OUT_DIR");
```

Comment:
Maybe this should go into a separate patch, it was helpful for debugging this, because `env!` caches the environment variables in the static section of the binary, so they can be run outside of cargo, like when running under a debugger.

Reply:
Let's leave it as-is in your PR: if it breaks, we'll find out at some point!